### PR TITLE
plugins: add Debian packaging for PostgreSQL plugin

### PR DESCRIPTION
**Pull request by joergsteffens**
Add Debian packaging for bareos-filedaemon-postgresql-python-plugin.
python*-dateutil is required for bareos-filedaemon-postgresql-python-plugin.
Adding the requirement to the packages bareos-filedaemon-python2-plugin
and bareos-filedaemon-python3-plugin,
as these handle the different type versions
and it is also required by other Python modules (e.g. libcloud).

### CODE REVIEW

File: debian/control.bareos-filedaemon-python-plugins-common [hunk 1]
```diff
@@ -24,18 +24,32 @@ Package:        bareos-filedaemon-percona-xtrabackup-python-plugin
 Architecture:   any
 Section:        python
 Pre-Depends:    debconf (>= 1.4.30) | debconf-2.0
-Depends:        bareos-common (= ${binary:Version}), bareos-filedaemon-python-plugin (= ${binary:Version})${misc:Depends}
+Depends:        bareos-common (= ${binary:Version}), bareos-filedaemon-python-plugin (= ${binary:Version}), ${misc:Depends}
+Recommends:     bareos-filedaemon-python3-plugin
 Description: Backup Archiving Recovery Open Sourced - file daemon Percona XtraBackup plugin
  Bareos is a set of programs to manage backup, recovery and verification of
  data across a network of computers of different kinds.
  .
```
Comment (fbergkemper) [inline debian/control.bareos-filedaemon-python-plugins-common:32]: unnecessary full stop

File: debian/control.bareos-filedaemon-python-plugins-common [hunk 2]
```diff
@@ -24,18 +24,32 @@ Package:        bareos-filedaemon-percona-xtrabackup-python-plugin
...
+Package:        bareos-filedaemon-postgresql-python-plugin
+Architecture:   any
+Section:        python
+Pre-Depends:    debconf (>= 1.4.30) | debconf-2.0
+Depends:        bareos-common (= ${binary:Version}), bareos-filedaemon-python-plugin (= ${binary:Version}), ${misc:Depends}
+Recommends:     bareos-filedaemon-python3-plugin
+Description: Backup Archiving Recovery Open Sourced - file daemon PostgreSQL plugin
+ Bareos is a set of programs to manage backup, recovery and verification of
+ data across a network of computers of different kinds.
+ .
```
Comment (fbergkemper) [inline debian/control.bareos-filedaemon-python-plugins-common:44]: unnecessary full stop

File: debian/control [hunk 3]
```diff
@@ -332,5 +332,6 @@ Description: Backup Archiving Recovery Open Sourced - common tools
 
 
 
+
```
Comment (fbergkemper) [inline debian/control:335]: I guess, debian/control should not be part of the PR.

File: docs/manuals/source/Appendix/Howtos.rst [hunk 4]
```diff
@@ -273,8 +275,8 @@ At least you gain a full backup which contains the follow:
 
 
 
-| So if you perform your first full backup your are capable to perfom differntial and incremental backups.
-| Differntial FileSet example:
+So if you perform your first full backup your are capable to perform differntial and incremental backups.
```
Comment (fbergkemper) [inline docs/manuals/source/Appendix/Howtos.rst:278]:
```suggestion
After your first full backup you will be able to also run differential and incremental backups.
```

File: debian/control (hunk 3 above)
Comment (joergsteffens) [reply debian/control:335]: In fact, I think this change is required.

**Review by fbergkemper — APPROVED** [review]
Looks good so far.
